tests/openassetio/test_imports.py

```diff
@@ -0,0 +1,130 @@
+#
+#   Copyright 2013-2021 [The Foundry Visionmongers Ltd]
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+"""
+These test ensure that all modules are importable. This can help catch
+dependencies between packages due to the hoisting required to hide
+duplicate namespaces to match the future C++ implementation appearance.
+"""
+
+import sys
+import pytest
+
+# pylint: disable=no-self-use
```

Comment:
[consider][minor] Multiple checks can be comma-separated after the `disable=` to avoid so many lines of code.

Reply:
Good idea, updated [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/123/commits/d37a29b404890fa441a056eda90a7de75fbe1602).